### PR TITLE
SERVICE: Constant TimeHelper

### LINKED ISSUE #42: SERVICE: CONSTAN
CREATED TimeHelper

### PR DESCRIPTION
**Pull request by rustambekovich**
close #42

### CODE REVIEW

File: src/GreenSaleuz.Service/Helpers/TimeHelper.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+namespace GreenSaleuz.Service.Helpers;
+
+public class TimeHelper
+{
+    public static DateTime GetDateTime()
+    {
+        var time = DateTime.UtcNow;
+        time = time.AddHours(TimeConstant.UTC);
+        return time;
```
Comment (SamandarYigitaliyev) [inline src/GreenSaleuz.Service/Helpers/TimeHelper.cs:11]: return dan oldin enter tashlash kerak

**Review by SamandarYigitaliyev — COMMENTED** [review]
To'g'irlab qayta yuklang

**Review by SamandarYigitaliyev — CHANGES_REQUESTED** [review]
Clean code ga amal qling

**Review by MrOOI — APPROVED** [review]

**Review by MrOOI — APPROVED** [review]

**Review by SamandarYigitaliyev — APPROVED** [review]
To'g'irlanibdi

**Comment by MrOOI** [comment]
Closes #49

**Pull request merged**